images/scripts/rhel.setup

```diff
@@ -173,6 +173,7 @@ echo foobar | passwd --stdin root
 # them to not spontaneously change from one test run to the next when
 # the distribution repository is updated.
 COCKPIT_DEPS="\
+criu \
```

Comment:
cockpit-podman doesn't actually requires/recommend `criu`, so this would better fit in TEST_PACKAGES.